<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPage</class>
 <widget class="QWidget" name="ConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white;background:#153152</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>361</width>
     <height>469</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>361</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>361</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="title">
    <string>Config</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="NaviMapSet1Btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background:#335F90</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>image/QYD_2.png</normaloff>image/QYD_2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NaviMapSet2Btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background:#335F90</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>image/QYD_1.png</normaloff>image/QYD_1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NaviMapSet3Btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background:#335F90</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>image/SDG.png</normaloff>image/SDG.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NaviMapSet4Btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background:#335F90</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>image/HSG.png</normaloff>image/HSG.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NaviMapSet5Btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;background:#335F90</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>image/3rdQB.png</normaloff>image/3rdQB.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="NaviMapSet6Btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;background:#335F90</string>
            </property>
            <property name="text">
             <string>OPG</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTextEdit" name="txtMsgDisp">
            <property name="minimumSize">
             <size>
              <width>341</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>341</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
